Sources/Sentry/Public/SentryGeo.h

```diff
@@ -0,0 +1,35 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Geo)
+@interface SentryGeo : NSObject <SentrySerializable, NSCopying>
+
+/**
+ * Optional: Human readable city name.
+ */
+@property (nullable, atomic, copy) NSString *city;
+
+/**
+ * Optional: Two-letter country code (ISO 3166-1 alpha-2).
+ */
+@property (nullable, atomic, copy) NSString *countryCode;
```

Comment:
I wonder if we can't just grab the `NSLocale` object that would probably be used to populate this, so we can get any other info from that we need at a later date, as well as ensure the right value is actually set in our `serialize` implementation (there's a lot going on in NSLocale and some people might not get it right).

Reply:
As i understood this is filled out manually by users, so i'm not sure if we should do things automatically to populate these fields. What other info that we might need are you refering to?

Reply:
I suppose even if we know sure we won't want anything other than country code, we could still just get it directly from the NSLocale. I've seen plenty of mistakes mixing up country codes, language codes and locale identifiers. I would ask for some validation logic, but that would be more work than us just using the locale object in a known safe way.

> this is filled out manually by users

Do you mean end-users of apps consuming this SDK? Or users of the SDK, as in app developers? If the former, even more reason to validate input, but again, that's more work.

Reply:
the server infers that automatically from the IP if provided, this should only be set if the user wanna override the inferred value by the server as explained here https://github.com/getsentry/sentry-cocoa/pull/2710#discussion_r1127830155